ext/mbstring/tests/illformed_utf_sequences.phpt

```diff
@@ -16,24 +16,23 @@ function chk_enc($str, $n, $enc = "UTF-8") {
 
 mb_substitute_character(0xfffd);
 
-
 echo "UTF-8 redundancy\n";
```

Comment:
What's up with the changes in this file?

Reply:
Okay, I guess the new implementation has different error handling. I think it would be good if both implementations stay in sync though -- could we first change the old implementation to the new error handling behavior in a separate PR, so that this one does not contain functional changes?

Reply:
Well, maybe... if you twist my arm 😆

Reply:
I'm doing it the other way around: modifying the new code so it always produces exactly the same number of error markers as the old code. It's almost comical how complicated it is to do that.

Will follow up with a separate commit which changes the error handling behavior.

Reply:
Error handling behavior now matches the old version.